StatBot/Models/DataModel.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace StatBot.Models
 {
-    public class DataModel
+    public static class DataModel
     {
         private static List<User> _users;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the '_users' field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1450)